atomic_reactor/plugins/post_koji_upload.py

```diff
@@ -448,6 +449,19 @@ def add_log_type(output, arch):
         image = add_buildroot_id(output)
         output_files.append(image)
 
+        # add operator manifests to output
+        operator_manifests_path = (self.workflow.postbuild_results
+                                   .get(PLUGIN_EXPORT_OPERATOR_MANIFESTS_KEY))
+        if operator_manifests_path:
+            operator_manifests_file = open(operator_manifests_path)
+            manifests_metadata = self.get_output_metadata(operator_manifests_path,
+                                                          'operator_manifests.zip')
+            operator_manifests_output = Output(file=operator_manifests_file,
+                                               metadata=manifests_metadata)
+            operator_manifests_output.metadata.update({'arch': arch, 'type': 'log'})
```

Comment:
Could put a comment here explaining that 'log' is used as a stop-gap until a more appropriate type name is supported by Koji.